### DISCUSSION TITLE
Mouse Control Inconsistency?

### DISCUSSION THREAD

**Discussion by Jerahya** [barry-ran/QtScrcpy#489] [Q&A]
After awhile of playing COD mobile I notice the mouse control lost consistency. The mouse control is stops working for a second and most of the tile inconsistent input which make it like I have way lower latency. Also it has a tendency for the direction key to get stuck where it will just set it to autorun and I can't do anything about it once It keeps running.

**Comment by pepijawa** [comment]
same, I played apex legends mobile and experienced something like this, solution is in our device , lower the graphic on the game.
cmiiw
